packages/foreman/foreman-selinux/foreman-selinux.spec

```diff
@@ -18,14 +18,7 @@
 
 %define selinux_variants targeted
 %define selinux_modules foreman foreman-proxy
-
-%if 0%{?rhel} == 5
-# absolute minimum versions for RHEL 5
-%define selinux_policy_ver 3.11.1-81
-%else
-# absolute minimum versions for RHEL 6
-%define selinux_policy_ver 2.4.6-80
-%endif
+%define selinux_policy_ver 3.13.1-192
```

Comment:
Pulp detects it automatically: `%global selinux_policyver %(rpm --qf "%%{version}-%%{release}" -q selinux-policy)`

https://github.com/pulp/pulp-packaging/blob/master/packages/pulp/pulp.spec#L20

Probably a better way forward.

Reply:
Nice, amended. I also removed the policycoreutils version, it was probably for RHEL5, this was copy*paste from Satellite5 packaging.